extensions/ql-vscode/src/view/model-editor/ModelAlertsIndicator.tsx

```diff
@@ -27,6 +28,9 @@ export const ModelAlertsIndicator = ({
     return null;
   }
 
+  const revealInModelAlertsView = () =>
```

Comment:
I'd just inline the code from `sendRevealInModelAlertsViewMessage` instead of having a separate function. Removes an unnecessary level of indirection.